airgun/views/host_new.py

```diff
@@ -795,6 +795,26 @@ class EditSystemPurposeView(View):
     cancel = OUIAButton('cancel-syspurpose')
 
 
+class ManageHostStatusesView(View):
+    """Manage host statuses modal"""
+
+    ROOT = './/div[@data-ouia-component-id="statuses-modal"]'
+
+    host_statuses_table = Table(
+        locator='.//table[contains(@class, "pf-c-table")]',
```

Comment:
Is it possible to use ouia-id(`data-ouia-component-id="statuses-table"`) here as a locator? And https://github.com/RedHatQE/widgetastic.patternfly4/blob/master/src/widgetastic_patternfly4/ouia.py#L97C7-L97C22